### DISCUSSION TITLE
Using DrawTextLayout after 0.8.1 update with DirectWriteCore creating TextLayout

### DISCUSSION THREAD

**Discussion by robhurry** [microsoft/WindowsAppSDK#1108] [Q&A]
After updating to 0.8.1 from 0.8.0 my application that I had successfully transfered to the windows app sdk because the DrawTextLayout to the ID2D1DeviceContext now fails. 

I see from the documentation of DWriteCore that (https://docs.microsoft.com/en-us/windows/win32/directwrite/dwritecore-overview):
"However, currently, Direct2D and DWriteCore don't interoperate. For example, if you create an IDWriteTextLayout using DWriteCore, and pass it to D2D1RenderTarget::DrawTextLayout, then that call will fail."

I assume this explains why it no longer works in version 0.8.1 (Because the DirectWriteCore package is now added) so I assume my IDWriteTextLayout is created using DWriteCore.

So my question is in a Windows App SDK app using 0.8.1 how do you draw a text layout to a D2D1RenderTarget?

**Comment by supthos** [comment]
They've added a render target interface that allows you to copy the image data, but still, it's been how many years later, and you have nowhere to copy it into, so you could assign it to a xaml image. It's a shame. 
Who are the developers in charge of this repo anyway? What are they being paid for it? Cause they need to do a better job.
